backend/hitas/services/owner.py

```diff
@@ -114,3 +114,24 @@ def find_owners_with_multiple_ownerships() -> list[OwnershipWithApartmentCount]:
             "sale__apartment__street_address",
         )
     )
+
+
+def find_apartments_by_housing_company(housing_company_id: int) -> list[OwnershipForHousingReport]:
+    return (
+        Ownership.objects.filter(
+            sale__apartment__building__real_estate__housing_company_id=housing_company_id,
+            sale__apartment__building__real_estate__housing_company__regulation_status=RegulationStatus.REGULATED,
+        )
+        .select_related("sale__apartment")
+        .annotate(
+            apartment_number=F("sale__apartment__apartment_number"),
+            apartment_surface_area=F("sale__apartment__surface_area"),
+            apartment_share_number_start=F("sale__apartment__share_number_start"),
+            apartment_share_number_end=F("sale__apartment__share_number_end"),
+            sale_purchase_date=F("sale__purchase_date"),
+            owner_name=F("owner__name"),
+            owner_identifier=F("owner__identifier"),
+            owner_non_disclosure=F("owner__non_disclosure"),
+        )
```

Comment:
Why are we annotating these values, instead of just accessing them wherever needer?